### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by leolpaz**
### Leaderboard - Setup

Added on this pull request:

- Webpack setup.
- Update to ReadMe file.
- HTML markup.
- Minimal styling to follow the given wireframe.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './style.css';
```
Comment (helman101) [inline src/index.js]: - [x] At this point of the project you should have developed a working version of the application as the project requirements said. Kindly take a look at this screenshot and to project requirements. 😸 
    - All the buttons should be functional, which means, the submit button should add a score to the list, and the refresh button should refresh the list of scores 🚀
    - Remember to use ES6 syntax
    imagen

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🔴 

Hi @leolpaz  👋, 

You have done a great job with the template of your page,  👏👏👏 
but there are still some changes that you have to do.

## To Highlight 🌟 

- Nice job on the page template 😄 
- Nice job setting up webpack 🚀 

## Required Changes ♻️

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (leolpaz) [reply src/index.js]: Thanks! I got caught on the requirements and overlooked the description, will get right on fixing it.

Edit: fixed.

**Review by carlos-ssh — APPROVED** [review]
Hi @leolpaz , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**